app/assets/stylesheets/adoption_applications.css.scss

```diff
@@ -0,0 +1,56 @@
+h1 {
+  font-family: 'Architects Daughter', cursive;
+  padding-top: 30px;
+}
+
+.form_questions {
+  @include span-columns(8);
+  @include shift(2);
+}
+
+.radio {
+  padding: 10px 0;
+}
+
+.note {
+  @include span-columns(8);
+  @include shift(0);
+  color: $dark-blue;
+  padding: 10px 0 20px;
+  text-align: center;
+  width: 100%;
+}
+
+.field-with-errors {
+  input {
+    background: $error-color;
+  }
+
+  label {
+    color: darken($error-color, 60%);
+  }
+}
+
+#error_explanation {
```

Comment:
Avoid using id selectors<br>Selector `error_explanation` should be written in lowercase with hyphens